app/controllers/stories_controller.rb

```diff
@@ -69,7 +69,7 @@ def assign_hero_html
 
   def get_latest_campaign_articles
     campaign_articles_scope = Article.tagged_with(SiteConfig.campaign_featured_tags, any: true).
-      where("published_at > ?", 2.weeks.ago).where(approved: true).
+      where("published_at > ?", 4.weeks.ago).where(approved: true).
```

Comment:
I need to circle back on #7582 but this temporarily ensures we're showing the right posts.